update-instructions.md

```diff
@@ -102,6 +102,16 @@ If there is a new Minecraft version:
 
 </details>
 
+### GDLauncher Carbon
+
+1. Select existing FO instance
+2. Go to the `Settings` tab
+3. Click `⇄ Change Modpack Version`
+4. Select the latest version from the dropdown
+5. Click `Update`
+
+**Note**: the instance name will not update, so it may seem outdated. You'll always find the correct version in the details page or bottom right corner of the game's title screen.
```

Comment:
Despite this note, as of v2.0.0-beta.1709163930 the upgrading system still seems flawed, 5.6.5 -> 5.8.0-beta.11 and 5.8.0-beta.10 -> 5.8.0-beta.11 updates did not occur smoothly for me (some parts remained outdated). Need to check that upon release and not merge this until it is fixed.